README.md

```diff
@@ -66,6 +66,7 @@ Then it can be included in this list!
 | [Synthizer] | 16 | 🌙 Inactive | A simple, experimental functional language for real time additive audio synthesis. |
 | [Radicle] | 10 | 🌙 Inactive | An Ur-lisp interpreter written in Rust. |
 | [Lang] | 7 | 🌙 Inactive | An imperative programming language written in Rust. |
+| [Nyson] | 6 | ☀️ Active | A simple, fast, scripting language that can compile made in rust |
```

Comment:
Thanks for the submission! Could you relocate this line to the right spot? The sorting on the table is by activity first (active on top, inactive on bottom) and then by star count.